### ISSUE TITLE
Feature Request: Persistent Borders

### ISSUE THREAD

**Issue by Mirage-Panda** [FelixKratz/JankyBorders#131]
I would like to request a toggle that would keep JankyBorders visible on neighbouring spaces. So during the space switch animation, instead of the borders loading in **after** focusing the space, it would already be visible, making the space switch animations look much more polished.
Ideally, there should be an option like `visible_neighbouring_borders` to configure it.
For example, `visible_neighbouring_borders=1` would look something like this:
```
+-----Space 1-----+ +-----Space 2-----+ +-----Space 3-----+ +-----Space 4-----+ +-----Space 5-----+
|                 | |                 | |                 | |                 | |                 |
|borders unloaded | | borders loaded  | |  current space  | | borders loaded  | |borders unloaded |
|                 | |                 | |                 | |                 | |                 |
+-----Space 1-----+ +-----Space 2-----+ +-----Space 3-----+ +-----Space 4-----+ +-----Space 5-----+
```
Below is a video showing the awkward transition of the space switch animation

https://github.com/user-attachments/assets/87fb4347-569b-4a54-bd3b-4bd7198c458f

**Comment by FelixKratz** [comment]
Yes I think that should be the expected behaviour. It likely is only a small tweak in the code. I never noticed it because I have those animations disabled.

**Comment by Mirage-Panda** [comment]
I was wondering if the rendered borders doesn't cause much performance issues, is it possible to make this configurable (either boolean or an integer)? Thanks!

**Comment by tristanoneil** [comment]
Maybe worth noting that the borders disappear when you trigger mission control as well.

**Comment by Mirage-Panda** [comment]
> Maybe worth noting that the borders disappear when you trigger mission control as well.

Not really, I think they just go behind the windows

**Comment by Bellavene** [comment]
> Maybe worth noting that the borders disappear when you trigger mission control as well.
That is an expected behavior, mission control has its own borders when you select a window. And I believe you can't hardcode borders to an external app so it will be shown in mission control without too much of useless effort. It shows raw applications after all.

**Comment by Mirage-Panda** [comment]
@FelixKratz Hi, just following up on the issue earlier, I know you mentioned it was just a small tweak—is there any progress on this recently?
